fabric/src/main/resources/architectury.mixins.json

```
@@ -2,8 +2,8 @@
   "required": true,
   "package": "dev.architectury.mixin.fabric",
   "plugin": "dev.architectury.plugin.fabric.ArchitecturyMixinPlugin",
-  "compatibilityLevel": "JAVA_16",
-  "minVersion": "0.7.11",
+  "compatibilityLevel": "JAVA_167",
```

At least this change should be wrong
```suggestion
  "compatibilityLevel": "JAVA_17",
```

---

lol my bad